Configuration.md

```diff
@@ -15,71 +36,74 @@ opt += val4 --> opt contains val1, val2, val3 and val4
+On startup, ```autopin+``` accepts configuration options from two
+different sources:
 
-    A default configuration file with standard options can be loaded by setting the environment variable ```AUTOPIN_DEFAULT_CONFIG``` to the path of the desired configuration file.
+  - Process configuration
 
-  - User configuration
+    The process configuration can be supplied by the commandline
+    switch ```-c``` or the long option```--config```. There can be
+    multiple process configurations.
 
-    By starting ```autopin+``` with the argument ```-c <user_config>``` where ```<user_config>``` is the path to a configuration file, one can provide more specific configuration options.
+  - Global configuration
 
-  - Command line
+    Autopin+ can optionally supplied with a global configuration file.
+    It is used for configuring global options.
 
-    Configuration options can also be specified directly on the command line. In this case the prefix ```--``` must be added to the corresponding option:
-
-    ```
-    autopin+ -c user_config --opta+=val4 --optb=val8
-    ````
-
-The three configuration sources are loaded in the order shown above. This means that the user configuration can overwrite settings specified in the default configuration. Command line options can overwrite both the user configuration and the default configuration.
-
-# Basic configuration options
+## Process configuration options
 
 The following options are available:
 
   - ```Exec = <string>``` (no default)
 
-    ```autopin+``` will start the observed process with the command specified in the argument.
+    ```autopin+``` will start the observed process with the command
+    specified in the argument.
 
   - ```Attach = <string>|<int>``` (no default)
 
-    ```autopin+``` will attach to the process with the given name or the given pid.
-
-  - ```Trace = <boolean>``` (defaults to ```false```)
-
-    Enable or disable process tracing.
+    ```autopin+``` will attach to the process with the given name or
+    the given pid.
 
   - ```CommChan = <string>|<boolean>``` (no default).
 
-    Enable the communication channel for notifications from the observed process. If the argument is a string it is interpreted as the address for the communication channel. If no address is specified but the argument is set to true ```autopin+``` will use a default address.
+    Enable the communication channel for notifications from the
+    observed process. If the argument is a string it is interpreted as
+    the address for the communication channel. If no address is
+    specified but the argument is set to true ```autopin+``` will use
+    a default address.
 
   - ```Logfile = <string>``` (no default)
 
-    If this option is set, the output of ```autopin+``` will be redirected to the file specified in the argument.
+    If this option is set, the output of ```autopin+``` will be
+    redirected to the file specified in the argument.
 
   - ```PinningHistory.load = <string>``` (no default)
```

Comment:
Please remove PinningHistory.* options altogether. Sorry, I should have done this before.